pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -285,6 +287,321 @@ profiles:
 apiVersion: kubescheduler.config.k8s.io/v1beta1
 kind: KubeSchedulerConfiguration
 profiles:
+- pluginConfig:
+  - name: DefaultPreemption
+    args:
+  - name: InterPodAffinity
+    args:
+  - name: NodeResourcesFit
+  - name: OutOfTreePlugin
+    args:
+  - name: NodeResourcesLeastAllocated
+    args:
+  - name: NodeResourcesMostAllocated
+    args:
+  - name: VolumeBinding
+    args:
+  - name: PodTopologySpread
+  - name: NodeAffinity
+`),
+			wantProfiles: []config.KubeSchedulerProfile{
```

Comment:
the delta here is hard to see... were there any changes in expected results from loading v1beta1 configs?

Reply:
Nope, all changes are restricted to s/v1beta1/v1beta2. Diff of this test: http://ix.io/2RjD